packages/safety-suite-api/src/api/dashboardSink/fleetPerfomanceDashboard.ts

```diff
@@ -0,0 +1,335 @@
+import {Request} from '@idelic/safety-net';
+
+import {runApi} from '../../runApi';
+import {ApiOptions, ApiSuccessResponse} from '../../types';
+
+interface Sort {
+  column: string;
+  direction?: string;
+}
+
+interface Query {
+  customerAlias: string;
+  groupIds?: string[];
+}
+
+interface TableQuery extends Query {
+  sort: Sort;
```

Comment:
`Sort[]`.  You can provide multiple sorts.  Order matters.  Frontend will probably just use one sort for now, but we the backend supports multiple sorts for free so we have it for when we want it.